stats.go

```diff
@@ -112,6 +115,51 @@ func startPrometheusStats(l *logrus.Logger, i time.Duration, c *config.C, buildV
 	pr.MustRegister(g)
 	g.Set(1)
 
+	// Export the start time of the current binary execution
+	start_time := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   namespace,
+		Subsystem:   subsystem,
+		Name:        "start_time",
+		Help:        "Time the nebula client started, in unixtime",
+		ConstLabels: prometheus.Labels{},
+	})
+	pr.MustRegister(start_time)
+	start_time.Set(float64(time.Now().Unix()))
+
+	// Export our certificate information as labels on a static gauge
+	for _, ca := range caCert.CAs {
+
+		ca_info := prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "ca_cert_expiry",
+			Help:      "Returns CA cert time expiry in unixtime",
+			ConstLabels: prometheus.Labels{
+				// TODO : fill these in with actual data, not hardcoded fields
+				"name":       ca.Details.Name,
+				"public_key": hex.EncodeToString(ca.Details.PublicKey),
```

Comment:
We should use `clientCert.Sha256Sum()` instead since this is how the `pki.blocklist` works what is printed in many of the cert related logs.